system/smbus.c

```diff
@@ -59,18 +43,11 @@ static uint8_t nf_read_spd_byte(uint8_t smbus_adr, uint8_t spd_adr);
 static uint8_t ali_m1563_read_spd_byte(uint8_t smbus_adr, uint8_t spd_adr);
 static uint8_t ali_m1543_read_spd_byte(uint8_t smbus_adr, uint8_t spd_adr);
 
-static inline uint8_t bcd_to_ui8(uint8_t bcd)
-{
-    return bcd - 6 * (bcd >> 4);
-}
-
 void print_smbus_startup_info(void)
```

Comment:
`print_spd_startup_info()`, as suggested by Sam in #410 , so that the caller doesn't need to use preprocessor conditionals to select the function according to the ISA :)

Reply:
Ah, sure, will do.

Overall, I tried to limit the scope of my changes, expecting it would be done as a part of https://github.com/memtest86plus/memtest86plus/pull/410, but happy to handle it here.